app/Http/Controllers/QrCodeController.php

```diff
@@ -26,17 +26,22 @@ public function __invoke(Request $request): StreamedResponse
         $dark = [3, 7, 18, 100];
         $bgColor = $request->query('theme') === 'light' ? $light : $dark;
 
-        $qrCode = $qrCodeGenerator
-            ->margin(2)
-            ->size(512)
-            ->format('png')
-            ->backgroundColor(...$bgColor)
-            ->color(236, 72, 153, 100)
-            ->merge('/public/img/ico.png')
-            ->errorCorrection('M')
-            ->generate(route('profile.show', [
-                'username' => $user->username,
-            ]));
+        try {
+            $qrCode = $qrCodeGenerator
+                ->margin(2)
+                ->size(512)
+                ->format('png')
+                ->backgroundColor(...$bgColor)
+                ->color(236, 72, 153, 100)
+                ->merge('/public/img/ico.png')
+                ->errorCorrection('M')
+                ->generate(route('profile.show', [
+                    'username' => $user->username,
+                ]));
+        } catch (\Exception $e) {
```

Comment:
can you show any case? (I mean if exception is handled how we gonna know that it has error?
we must know what exception it can throw.